nova/core/requirements/requirements.yml

```
@@ -1,27 +1,21 @@
 ---
 roles:
   - name: geerlingguy.apache
-    src: https://github.com/geerlingguy/ansible-role-apache.git
     version: 3.3.0
   - name: geerlingguy.composer
-    src: https://github.com/geerlingguy/ansible-role-composer.git
     version: 1.9.2
   - name: geerlingguy.mysql
-    src: https://github.com/geerlingguy/ansible-role-mysql.git
     version: 4.3.3
```

```suggestion
    version: 4.3.4
```

Bumping this to `4.3.4` [(diff)](https://github.com/geerlingguy/ansible-role-mysql/compare/4.3.3...4.3.4) would include a fix for https://github.com/ansible/ansible/issues/82264  and should result in no breaking changes